License memes! (Updates readme.md to match changelog header + other information)

skull132 opened the pull request:
@Neerti brought up a fun point on Discord. The licensing of assets in the Baystation12 codebase is ambiguous. Unlike the [/tg/station README](https://github.com/tgstation/tgstation/#license), which is clear as day.

Things amended:
* tgUI license notification added, as per [this file](https://github.com/Baystation12/Baystation12/blob/dev/tgui/LICENSE.md).
* Clarifies that all assets/content/w/e is licensed under CC-BY-SA 3.0, as [per line 29 of the changelog header](https://github.com/Baystation12/Baystation12/blob/dev/html/templates/header.html#L29). Anything else would create a mahoosive incompatibility with the /tg/ codebase, as CC-BY-SA 3.0 and AGPL are in no way compatible with eachother. (I guess you're free to pursue the intent that **all** content is licensed under AGPL3, but then all the icons already licensed under it would be incompatible and aaaaaaaaaaaaaaaa. (Oh and please actually update the header to reflect that then.))

Why:
Because clarity is important and questions regarding this stuff should be minimal.

PsiOmegaDelta commented:
The added this-is-licensed-under phrasing isn't consistent with the exiting language, but I suppose that's minor nitpicking.

However, similar to Polaris you should further clarify what is covered by AGPL. For reference see https://github.com/PolarisSS13/Polaris/pull/3672/files (note the additions and replacements of words with "code").

skull132 commented:
Donezinod.

skull132 committed:
Update licensing information in the readme.md
Adds clarity to it and makes it work out properly with other information in the codebase.

skull132 commented:
\> Not using squash-merge.

:ree:
